<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPage</class>
 <widget class="QWidget" name="TabPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editIp"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPort"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbbProtocl">
       <item>
        <property name="text">
         <string>Tcp Server</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tcp Client</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Udp</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
